### PR TITLE
Add MIOTA support

### LINKED ISSUE #116: Add support for MIOTA addresses in encoding library
Here is the encoding library: https://github.com/ensdomains/address-encoder

Example: https://github.com/ensdomains/address-encoder/pull/47/files

Discord for dev help: https://discord.com/invite/AskZbFx

Please provide a reference link so we can check you did the encoding correctly.

We'd like to keep the encoding library as small as possible, so please be efficient in your use of space.

### PR DESCRIPTION
**Pull request by canokaue**
https://gitcoin.co/issue/ensdomains/address-encoder/116/100023814

## Issue number

Closes #116

## Description

Add MIOTA support.

## List of features added/changed

Add MIOTA on:
src/index.ts
src/tests/index.test.ts
README.md
Already sorted by Cointype.

NOTE: If you are adding new coin address support, please make sure to add a reference link to README so that reviewer can verify.
https://github.com/satoshilabs/slips/blob/master/slip-0044.md
https://github.com/Thoralf-M/IOTA-Ed25519-addresses

## How Has This Been Tested?

Local test run
Discord Dev discussion
TravisCI

## Checklist:

- [x] My code follows the code style of this project.
- [x] My code implements all the required features.

### CODE REVIEW

**Comment by Arachnid** [comment]
I have updated the Travis build to check that entries in the formats, tests, and README lists are in the correct order. Please check your PR and update it to maintain correct ordering in these lists.

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ This library currently supports the following cryptocurrencies and address forma
  - ICX
  - KSM (ss58)
  - LTC (base58check P2PHK and P2SH, and bech32 segwit)
+ - MIOTA (base58check)
```
Comment (makoto) [inline README.md:52]: Can you change it to `IOTA` as that's what's in the slip 44?

Comment (canokaue) [reply README.md]: sure thing

**Comment by canokaue** [comment]
@makoto changed MIOTA into IOTA according to slip 44 and resolved small conflict.

**Pull request merged**